perllib/FixMyStreet/Cobrand/TfL.pm

```diff
@@ -127,6 +127,16 @@ sub problems_sql_restriction {
     return $q;
 }
 
+sub inactive_reports_filter {
+    my ($self, $time, $rs) = @_;
+    if ($time == 3*12) {
```

Comment:
This has the side-effect of selecting all reports unless `$time` is exactly`36` or `84`, which might cause issues if a manual one-off run used e.g. 24 months. Would it be better to have a specific flag for safety-critical reports, or treat anything less than `84` as non-safety critical?

Reply:
Yep, good idea, done, and added tests for everything now.